packages/cubejs-docker/cube.js

```diff
@@ -0,0 +1,45 @@
+const MsSQL = require('@cubejs-backend/mssql-driver');
+const Teradata = require('@cubejs-backend/teradata-jdbc-driver');
+const { TeradataDriver } = Teradata
+
+function mssql() {
+    return new MsSQL({
+        server:"192.168.120.239",
```

Comment:
We should remove all hard-coded information such as: 
- IP Addresses
- Host names
- Secrets: user names / passwords

and move it to an external env var (.env) similar to what we do with our fullstack projects.

We should probably add an .env.example file for simplicity and add instructions for setup in the README.
We should also check for the existence of these env vars and bail out (process.exit) on failures.

Reply:
All sensitive are moved to a .env file as requested.
but IMHO all the file should be in a .git.ignore and we do a cube.js.example.